piqueserver/scripts/geoip.py

```diff
@@ -1,27 +1,37 @@
-"""
+'''
 Gets a player's location info using a geoip database.
+Location is formatted as a list of regions ordered by hierarchy (i.e. City, Region, Country)
 
 .. note::
-  This script depends on `pygeoip` package and `piqueserver --update-geoip` needs to be executed after installing the package.
+  This script depends on `geoip2` package and `piqueserver --update-geoip` needs to be executed after installing the package.
 
 Commands
 ^^^^^^^^
+* ``/from ``         get active player's location
 * ``/from <player>`` get player's location info
 
 .. codeauthor:: ?
-"""
+
+Updates
+^^^^^^^
+    v1.2
+    Author: bieito98
+    Date: 13-01-2019
+    Comment: support for new GeoLite database format and API version (GeoLite2-City.mmdb & geoip2)
```

Comment:
the formatting here isn't valid, but aside from that, I wonder it really makes sense to list the changelog for these things on the individual scripts, as opposed to the main piqueserver changelog. Thoughts?

Reply:
You are right, deleted